support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -286,15 +286,15 @@ function sendEventABTestParticipations(participations: Participations): void {
 
 function addQM() {
 	return loadScript(
-		'https://cdn.quantummetric.com/instrumentation/1.31.5/quantum-gnm.js',
+		'https://cdn.quantummetric.com/instrumentation/1.31.5/quantum-gnmx.js',
 		{
 			async: true,
 			integrity:
 				'sha384-QqJrp8s9Nl3x7Z6sc9kQG5eYJLVWYwlEsvhjCukLSwFsWtK17WdC5whHVwSXQh1F',
 			crossOrigin: 'anonymous',
 		},
-	).catch((e: Error) => {
-		logException(`Failed to load Quantum Metric: ${e.message}`);
+	).catch(() => {
```

Comment:
This was logging "undefined" as e.message and the error it gets upon inspection isn't helpful so I'm removing it.